hyp3lib/get_dem.py

```diff
@@ -140,84 +135,6 @@ def get_tile_for(args):
                 shutil.copy(source_file, output_dir)
 
 
-def write_vrt(dem_proj, nodata, tile_list, poly_list, out_file):
```

Comment:
Rudi wrote the custom write_vrt, but I'm don't recall why.  You may want to see if he remembers the reason.

Reply:
Test some dateline cases and you will know why I wrote this version. The coverage shapefile contains extra tiles around the dateline. The standard GDAL version does not handle any of this properly in this context.